pdns/recursordist/reczones.cc

```diff
@@ -325,8 +325,8 @@ std::tuple<std::shared_ptr<SyncRes::domainmap_t>, std::shared_ptr<notifyset_t>>
       // headers.first=toCanonic("", headers.first);
       if (n == 0) {
         ad.d_rdForward = false;
-        SLOG(g_log << Logger::Error << "Parsing authoritative data for zone '" << headers.first << "' from file '" << headers.second << "'" << endl,
-             log->info(Logr::Error, "Parsing authoritative data from file", "zone", Logging::Loggable(headers.first), "file", Logging::Loggable(headers.second)));
+        SLOG(g_log << Logger::Info << "Parsing authoritative data for zone '" << headers.first << "' from file '" << headers.second << "'" << endl,
```

Comment:
I always find it hard to choose between Info and Notice, but in this case I would say Notice.
```
     LOG_NOTICE    Conditions that are not error conditions, but should possibly be
                   handled specially.

     LOG_INFO      Informational messages.
```